docs/webserver/overview.md

```diff
@@ -0,0 +1,17 @@
+# iRobot® Create® 3 Webserver Overview
+The Create® 3 robot runs a configuration webserver for modifying settings of the robot.
+This is a separate process from its ROS 2[^1] application; it is running regardless of the status of that application.
+The webserver can be accessed via the robot's `usb0` (Ethernet over USB host), `wlan0` (provisioned to an external access point), or `wlan1` (robot as its own access point) interfaces.
```

Comment:
Should you specifically call out the fixed IP addresses for the usb0 and wlan1 and mention that it will be at your robot's wifi IP on wlan0 (for the people with less networking understanding)